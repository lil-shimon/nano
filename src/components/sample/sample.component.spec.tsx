import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import { Sample } from "./sample.component";

describe("Sample", () => {
  it("snapshot", () => {
    const { asFragment } = render(<Sample />);
    expect(asFragment()).toMatchSnapshot();
  });

  it.todo("renders correctly");
});
