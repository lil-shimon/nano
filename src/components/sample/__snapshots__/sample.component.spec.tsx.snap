// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sample > snapshot 1`] = `
<DocumentFragment>
  <section>
    <h1>
      Sample Header
    </h1>
    <div>
      <p>
        Sample paragraph text.
      </p>
    </div>
  </section>
</DocumentFragment>
`;
